ereol_app_feeds/src/Feed/CategoriesFeed.php

```diff
@@ -62,10 +67,14 @@ public function getData() {
       }
 
       if (!empty($subcategories)) {
+        $wrapper = entity_metadata_wrapper('node', $node);
+
         $data[] = [
           'title' => $node->title,
           'type' => 'category',
-          'query' => ParagraphHelper::VALUE_NONE,
+          'query' => !empty($wrapper->field_category_query->value())
```

Comment:
Like it more as a one-liner

Reply:
Rewritten to `if (…) { … }` for better readability.